omnibus/config/software/datadog-agent-integrations-py2.rb

```diff
@@ -432,9 +432,12 @@
         end
 
         # Copy SNMP profiles
-        profiles = "#{check_dir}/datadog_checks/#{check}/data/profiles"
-        if File.exist? profiles
-          copy profiles, "#{check_conf_dir}/"
+        profile_folders = ['profiles', 'default_profiles']
+        profile_folders.each do |profile_folder|
+            folder_path = "#{check_dir}/datadog_checks/#{check}/data/#{profile_folder}"
```

Comment:
Just to be clear, we want to copy the content of `profiles` just because there is a ReadMe inside and we don't expect to ever be copying yaml files here, correct?

Reply:
@FlorianVeaux Yes, that's right. We copy `profiles` for the readme.md and for having a folder that is already created  as hint for users to know this is where should put their profiles.